SimPEG/MT/Problem1D/Probs.py

```diff
@@ -0,0 +1,265 @@
+from SimPEG.EM.Utils import omega
+from SimPEG import mkvc
+from scipy.constants import mu_0
+from SimPEG.MT.BaseMT import BaseMTProblem
+from SimPEG.MT.SurveyMT import Survey, Data
+from SimPEG.MT.FieldsMT import Fields1D_e
+from SimPEG.MT.Utils.MT1Danalytic import getEHfields
+import numpy as np
+import multiprocessing, sys, time
+
+
+class eForm_psField(BaseMTProblem):
+    """
+    A MT problem soving a e formulation and primary/secondary fields decomposion.
+
+    By eliminating the magnetic flux density using
+
+        .. math ::
+
+            \mathbf{b} = \\frac{1}{i \omega}\\left(-\mathbf{C} \mathbf{e} \\right)
+
+
+    we can write Maxwell's equations as a second order system in \\\(\\\mathbf{e}\\\) only:
+
+    .. math ::
+        \\left(\mathbf{C}^T \mathbf{M^f_{\mu^{-1}}} \mathbf{C} + i \omega \mathbf{M^e_\sigma}] \mathbf{e}_{s} =& i \omega \mathbf{M^e_{\delta \sigma}} \mathbf{e}_{p}
+    which we solve for \\\(\\\mathbf{e_s}\\\). The total field \\\mathbf{e}\\ = \\\mathbf{e_p}\\ + \\\mathbf{e_s}\\.
+
+    The primary field is estimated from a background model (commonly half space ).
+
+
+    """
+    # From FDEMproblem: Used to project the fields. Currently not used for MTproblem.
+    _fieldType = 'e_1d'
+    _eqLocs    = 'EF'
+    _sigmaPrimary = None
+
+
+    def __init__(self, mesh, **kwargs):
+        BaseMTProblem.__init__(self, mesh, **kwargs)
+        self.fieldsPair = Fields1D_e
+        # self._sigmaPrimary = sigmaPrimary
+
+    @property
+    def sigmaPrimary(self):
+        """
+        A background model, use for the calculation of the primary fields.
+
+        """
+        return self._sigmaPrimary
+    @sigmaPrimary.setter
+    def sigmaPrimary(self, val):
+        # Note: TODO add logic for val, make sure it is the correct size.
+        self._sigmaPrimary = val
+
+    def getA(self, freq):
+        """
+            Function to get the A matrix.
+
+            :param float freq: Frequency
+            :rtype: scipy.sparse.csr_matrix
+            :return: A
+        """
+
+        Mmui = self.mesh.getEdgeInnerProduct(1.0/mu_0)
```

Comment:
I would re-name this MeMui (lives on edges) and similarly for below, MfSigma (lives on faces) to be explicit